test/e2e/suite/conformance/certificates/tests.go

```diff
@@ -44,13 +44,20 @@ func (s *Suite) Define() {
 		ctx := context.Background()
 		f := framework.NewDefaultFramework("certificates")
 
-		// wrap this in a BeforeEach else flags will not have been parsed at
-		// the time that the `complete` function is called.
+		sharedIPAddress := "127.0.0.1"
+
 		BeforeEach(func() {
-			if !s.completed {
-				s.complete(f)
+			if s.completed {
+				return
+			}
+
+			s.complete(f)
+
+			if s.UseIngressIPAddress {
+				sharedIPAddress = f.Config.Addons.ACMEServer.IngressIP
 			}
```

Comment:
This doesn't look like it needs to be inside the `BeforeEach` and can instead live next to the `sharedIPAddress` declaration.